src/Store/VimStoreConnector.re

```diff
@@ -317,17 +319,59 @@ let start =
       open Vim.BufferUpdate;
       Log.info("Vim - Buffer update: " ++ string_of_int(update.id));
       open Core.Types;
+      open Model.State;
+
+      let isFull = update.endLine == (-1);
+
+      let bufferMaybe =
+        Some(getState())
+        |> Option.map(state => state.buffers)
+        |> Option.bind(Model.Buffers.getBuffer(update.id));
```

Comment:
Isn't this equivalent to

```reason
getState().buffers |> Model.Buffers.getBuffer(update.id);
```
?

Reply:
Sure is... don't know what I was thinking 😄